src/app/lib/analyticsUtils/chartbeat.js

```diff
@@ -0,0 +1,48 @@
+import Cookie from 'js-cookie';
+import onClient from '../utilities/onClient';
+
+const ID_COOKIE = 'ckns_sylphid';
+
+export const chartbeatUID = 50924;
+export const useCanonical = true;
+
+const buildSectionArr = (service, value, type) => [
+  `${service} - ${value}`,
+  `${service} - ${value} - ${type}`,
+];
+
+const buildServiceType = (service, type) => `${service} - ${type}`;
+
+const capitalize = s => s.charAt(0).toUpperCase() + s.slice(1);
```

Comment:
Wondered if we should start a new library for generic text utils to encourage reuse and reduce the chance of someone duplicating this elsewhere later? There's nothing Chartbeat-specific about it.

Reply:
I agree, something for this issue or a future one?

Reply:
Later is fine.